renderer/src/components/Sentry.tsx

```diff
@@ -4,6 +4,9 @@ import { BrowserTracing } from '@sentry/tracing'
 
 const SentryComponent = () => {
   useEffect(() => {
+    // Disable Sentry integration for dev builds
+    if (window.electron.stationBuildVersion.match(/\.1-dev$/)) { return }
```

Comment:
```suggestion
    if (window.electron.stationBuildVersion.endsWith('.1-dev')) { return }
```

This would be simpler, but then I wonder if `.1` is a meaningful segment to check for, and maybe we can just have this?

```suggestion
    if (window.electron.stationBuildVersion.endsWith('-dev')) { return }
```

Reply:
I agree `endsWith` is a better tool 👍🏻 

`.1` is part of the hard-coded build number `.1-dev`.

At the end of the day, this does not matter much. I am going with your second suggestion.